### ISSUE TITLE
ProjectSettings not needed?

### ISSUE THREAD

**Issue by AvinZarlez** [hengineer/CaptainsMess#1]
Since the point is to just copy the Captain's Mess files into your own Unity project, shouldn't the files in ProjectSettings not be tracked/included as part of the Git repo?

**Comment by hengineer** [comment]
Yes you're right. I'll remove it in the next update. If you know any best practices for organizing Unity extensions please let me know. When development is further along I'll make an official Unity asset package for the Asset Store.
